src/sentry/options/defaults.py

```diff
@@ -518,10 +518,6 @@
 register("sentry-metrics.cardinality-limiter.limits.releasehealth.per-org", default=[])
 register("sentry-metrics.cardinality-limiter.orgs-rollout-rate", default=0.0)
 
-
-# A rate to apply during ingest to turn on performance detection (just detection, no storage of events or issues)
-register("store.use-ingest-performance-detection-only", default=0.0)
```

Comment:
There are a few tests that also use this that we may as well clean up as well

Reply:
Ah true forgot about the event* options being used